README.md

```diff
@@ -22,6 +21,11 @@ The macOS 11.0 virtual environment is currently provided as a private preview on
 The "macos-latest" YAML workflow label still uses the macOS 10.15 virtual environment.
 ```
 
+```
+Ubuntu 16.04 is being deprecated. It is not recommended for new users. 

```

Comment:
May be put deprecation date here or attach announcement link https://github.blog/changelog/2021-04-29-github-actions-ubuntu-16-04-lts-virtual-environment-will-be-removed-on-september-20-2021/

Reply:
This link won't be clickable and will be hidden under the horizontal scrollbar, because formatting in code block works not very good. So I'd prefer to keep it as text only